src/commands/registerCommands.ts

```diff
@@ -47,6 +47,8 @@ export function registerCommands(): void {
     registerReportIssueCommand('azureResourceGroups.reportIssue');
     registerCommand('azureResourceGroups.createResource', createResource);
     registerCommand('azureResourceGroups.refreshWorkspace', refreshWorkspace);
+    registerCommand('azureWorkspace.loadMore', async (context: IActionContext, node: AzExtTreeItem) => await ext.workspaceTree.loadMore(node, context));
```

Comment:
None of the other workspace commands use the pattern of the prefix being `azureWorkspace`. I'm fine with it, but we should be consistent.

Reply:
Good point. I changed the other workspace command (refresh) to follow the same convention.